src/components/cc-addon-admin/cc-addon-admin.js

```diff
@@ -97,6 +105,8 @@ export class CcAddonAdmin extends LitElement {
 
     const isFormDisabled = this.error || this.saving;
     const loadingError = (this.error && !this.saving) || (this.error && this.addon == null);
+    const shouldShowVmText = !this.noDangerZoneVmText;
+    const shouldShowBackupsText = !this.noDangerZoneBackupText;
```

Comment:
suggestion: if we want to order the variables, this one should be above

Reply:
I chose to order them like that because VM Text goes before the Backups text within the render :man_shrugging: but to be honest, alphabetical order is good for me, I'll go with that :wink: